site/stripe/DonateForm.tsx

```diff
@@ -197,161 +202,137 @@ export class DonateForm extends React.Component {
-                                name="interval"
-                                onChange={() => this.setInterval("once")}
-                                checked={this.interval === "once"}
-                            />
-                            <label htmlFor="once">One time</label>
-                        </div>
-                        <div className="owid-radio">
-                            <input
-                                type="radio"
-                                id="monthly"
-                                value="monthly"
-                                name="interval"
-                                onChange={() => this.setInterval("monthly")}
-                                checked={this.interval === "monthly"}
-                            />
-                            <label htmlFor="monthly">Monthly</label>
-                        </div>
+                <fieldset>
+                    <legend className="overline-black-caps">Frequency</legend>
+                    <div className="donation-options">
+                        <input
+                            type="button"
+                            value="Give once"
+                            onClick={() => this.setInterval("once")}
+                            className={cx("donation-options__button", {
+                                active: this.interval === "once",
+                            })}
+                        />
+                        <input
+                            type="button"
+                            value="Monthly"
+                            onClick={() => this.setInterval("monthly")}
+                            className={cx("donation-options__button", {
+                                active: this.interval === "monthly",
+                            })}
+                        />
                     </div>
                 </fieldset>
 
-                <fieldset className="donate-form-currency">
-                    <legend>
-                        <h3>Currency</h3>
-                    </legend>
-                    <div className="owid-radios">
+                <fieldset>
+                    <legend>Currency</legend>
+                    <div className="donation-options">
                         {SUPPORTED_CURRENCY_CODES.map((code) => (
-                            <div key={code} className="owid-radio">
-                                <input
-                                    type="radio"
-                                    id={code}
-                                    value={code}
-                                    name="currency"
-                                    onChange={() => this.setCurrency(code)}
-                                    checked={this.currencyCode === code}
-                                />
-                                <label htmlFor={code}>{code}</label>
-                            </div>
+                            <input
+                                type="button"
```

Comment:
These should start to wrap on mobile probably (this is 360w - apparently the width of a Galaxy S20)

![image](https://github.com/owid/owid-grapher/assets/11844404/9477f2c6-454d-4587-9d6c-dd0e82b98af2)

Reply:
👍 Moving to a 2 col grid on small screens.